### PR TITLE
Readme: Fix the sample commands in the console section and other misc comments

### PR DESCRIPTION
**Pull request by gitsparty**
Fix console command line samples

### CODE REVIEW

File: dotnetapp-current/README.md [hunk 1]
```diff
@@ -28,14 +28,15 @@ Follow these steps to run this sample in a Linux container:
 dotnet restore
 dotnet publish -c Release -o out
 docker build -t dotnetapp .
-docker run dotnetapp Hello .NET Core from Docker
+docker run dotnetapp
 ```
 
-Follow these steps to run this sample in a  Windows container:
+Follow these steps to run this sample in a  Windows container.
+Note: Before doing the follwing, make sure you are running in "Windows Container" mode. You can do this by Right Clicking the Docker app (in the System Tray) -> "Switch to Windows Containers".
```
Comment (drewnoakes) [inline dotnetapp-current/README.md:35]: Small typo here (`follwing`).

File: dotnetapp-dev/README.md [hunk 2]
```diff
@@ -26,12 +26,13 @@ Follow these steps to run this sample in a Linux container:
 
 ```console
 docker build -t dotnetapp .
-docker run dotnetapp Hello .NET Core from Docker
+docker run dotnetapp
 ```
 
-Follow these steps to run this sample in a  Windows container:
+Follow these steps to run this sample in a  Windows container.
+Note: Before doing the follwing, make sure you are running in "Windows Container" mode. You can do this by Right Clicking the Docker app (in the System Tray) -> "Switch to Windows Containers".
```
Comment (drewnoakes) [inline dotnetapp-dev/README.md:33]: Ditto.

File: dotnetapp-prod/README.md [hunk 3]
```diff
@@ -28,14 +28,15 @@ Follow these steps to run this sample in a Linux container:
 dotnet restore
 dotnet publish -c Release -o out
 docker build -t dotnetapp .
-docker run dotnetapp Hello .NET Core from Docker
+docker run dotnetapp
 ```
 
-Follow these steps to run this sample in a  Windows container:
+Follow these steps to run this sample in a  Windows container. 
+Note: Before doing the follwing, make sure you are running in "Windows Container" mode. You can do this by Right Clicking the Docker app (in the System Tray) -> "Switch to Windows Containers".
```
Comment (drewnoakes) [inline dotnetapp-prod/README.md:35]: Ditto

File: dotnetapp-selfcontained/README.md [hunk 4]
```diff
@@ -54,23 +54,29 @@ Follow these steps to run this sample in a Linux container:
 dotnet restore
 dotnet publish -c Release -o out -r debian.8-x64
 docker build -t dotnetapp .
-docker run dotnetapp Hello .NET Core from Docker
+docker run dotnetapp
 ```
 
 Follow these steps to run this sample in a  Windows container:
+Note: Before doing the follwing, make sure you are running in "Windows Container" mode. You can do this by Right Clicking the Docker app (in the System Tray) -> "Switch to Windows Containers".
```
Comment (drewnoakes) [inline dotnetapp-selfcontained/README.md:61]: Ditto

File: dotnetapp-current/README.md [hunk 5]
```diff
@@ -28,14 +28,15 @@ Follow these steps to run this sample in a Linux container:
 dotnet restore
 dotnet publish -c Release -o out
 docker build -t dotnetapp .
-docker run dotnetapp Hello .NET Core from Docker
+docker run dotnetapp
 ```
 
-Follow these steps to run this sample in a  Windows container:
+Follow these steps to run this sample in a  Windows container.
```
Comment (MichaelSimons) [inline dotnetapp-current/README.md:34]: Extra space 'a  Windows'

File: dotnetapp-current/README.md (hunk 1 above)
Comment (MichaelSimons) [inline dotnetapp-current/README.md:35]: Consider adding a link to https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers for additional information.

File: dotnetapp-current/README.md [hunk 6]
```diff
@@ -19,7 +19,7 @@ Follow these steps to run the sample locally:
 
 ```console
 dotnet restore
-dotnet run Hello .NET Core from Docker
```
Comment (MichaelSimons) [inline dotnetapp-current/README.md:22]: While args are not required, the reason they were specified was to illustrate how you can pass args to the containerized app.

File: dotnetapp-selfcontained/README.md [hunk 7]
```diff
@@ -54,23 +54,29 @@ Follow these steps to run this sample in a Linux container:
...
+
+However from a runtime perspective, framework-dependent applications require the framework to be pre-installed on the machine, while self-contained applications include the entire framework in their deployment package. Both approaches have their obvious PROS and CONS.
 
 Self-contained apps are operating system- and chip-specific after they are published. The -r argument instructs the publish command to select the correct native assets to include. See [.NET Core Runtime IDentifier (RID) catalog](https://docs.microsoft.com/dotnet/articles/core/rid-catalog) for more information. 
 
-The `runtimes` node in project.json lists the suppored RIDs for a given app. A large set is listed in this sample to enable it to be uses on multiple operating systems. 
+The `runtimes` on which the app can run are listed in the RuntimeIdentifiers property in the CSPROJ. A large set is listed in this sample to enable it to be uses on multiple operating systems. For all possible runtime identifiers, see [.NET Core Runtime IDentifier (RID) catalog](https://docs.microsoft.com/dotnet/articles/core/rid-catalog).
+
+
+
```
Comment (MichaelSimons) [inline dotnetapp-selfcontained/README.md:82]: nit 😃 - Unnecessary trailing blank lines.

**Review by MichaelSimons — COMMENTED** [review]
Thanks for the contribution.  I just have a couple minor comments.

File: dotnetapp-current/README.md (hunk 6 above)
Comment (kendrahavens) [reply dotnetapp-current/README.md:22]: Yes, I think its a good idea to keep them in here.

File: dotnetapp-current/README.md (hunk 1 above)
Comment (kendrahavens) [reply dotnetapp-current/README.md:35]: These are great notes to add. Thank you!

**Review by kendrahavens — CHANGES_REQUESTED** [review]
Looks mostly good! Leave in the arguments for the `run` commands and fix the few typos and we'll be good to go.

**Comment by MichaelSimons** [comment]
@gitsparty, can you make the suggested edits or let us know any maybe someone else can.  Thanks

**Comment by MichaelSimons** [comment]
With multi-stage builds the experience has mostly been unified between Linux and Windows.  This PR is out of date.  Closing but feel free to make a new PR as appropriate if you still see issues.

**Pull request closed without merging**